### DISCUSSION TITLE
Anaconda Web UI installer break ventoy bootloader

### DISCUSSION THREAD

**Discussion by st-pathfinder** [rhinstaller/anaconda-webui#43] [General]
When installing through the new interface, the system installs successfully, but the bootloader on the flash drive breaks.
After ten installations of the system, the bootloader remained intact only once.
This only happens with the new interface.

**Comment by KKoukiou** [comment]
@st-pathfinder could you report this to bugzilla by following the instructions ehre https://anaconda-installer.readthedocs.io/en/latest/reporting.html ? Discussions section is meant for RFEs / Ideas, not for bug reports.

**Comment by KKoukiou** [comment]
Duplicate of: https://bugzilla.redhat.com/show_bug.cgi?id=2357173, it's fixed.
